Source/santad/EventProviders/SNTDeviceManager.h

```diff
@@ -0,0 +1,45 @@
+/// Copyright 2021 Google Inc. All rights reserved.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+///    Unless required by applicable law or agreed to in writing, software
+///    distributed under the License is distributed on an "AS IS" BASIS,
+///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///    See the License for the specific language governing permissions and
+///    limitations under the License.
+#import <DiskArbitration/DiskArbitration.h>
+#import <Foundation/Foundation.h>
+
+#include <EndpointSecurity/EndpointSecurity.h>
+
+@interface SNTDeviceManager : NSObject
+
+@property BOOL subscribed;
+@property BOOL blockUSBMount;
+@property NSArray<NSString *> *remountArgs;
+
+- (instancetype)init;
+
+- (void)listen;
+- (void)listenDA;
```

Comment:
Nit: would recommend not exposing `listenDA`/`listenES` externally

Reply:
Done